config/development.yaml

```diff
@@ -92,3 +92,7 @@ API_PROXY_CORS_ALLOWED: '*'
 
 # Current NYPL Webreader version
 READER_VERSION: v1
+
+# ANALYTICS CONFIGURATION
+DOWNLOAD_BUCKET: "ump-pdf-repository-logs"
+DOWNLOAD_LOG_PATH: "logs/946183545209/us-east-1/ump-pdf-repository/"
```

Comment:
nit: let's add a new line to the end of these yaml files so github stops complaining lol.